joomla/joomla-cms
# [5.2] Display the correct changelog when updating

Pull Request for Issue #43505 .

### Summary of Changes
When an extension provides a changelog URL in its update XML, a button is displayed to show the changelog, but the URL only loads the changelog from the current extension, not from the update site.


### Testing Instructions
1. Install an old version of a third party extension, for example a language pack.
2. Search for an update for that extension.
3. We now try to emulate a changelog available for that extension: Go to the database and in `#__updates` look for the entry for your extension update. Edit the changelogurl column and insert a URL to a `changelog.xml` in your sites root, for example `http://localhost/changelog.xml`
4. Create a `changelog.xml` with the content below. (Adapt the content to the extension you've been using for this)
5. Go to the "update extensions" view and see the button for the changelog for your extension.

### Content for the changelog.xml
```xml
<changelogs>
	<changelog>
		<element>pkg_ru-RU</element>
		<type>package</type>
		<version>5.2.2.1</version>
		<security>
			<item>Item A</item>
			<item><![CDATA[<h2>You MUST replace this file</h2>]]></item>
		</security>
		<fix>
			<item>Item A</item>
			<item>Item b</item>
		</fix>
	</changelog>
</changelogs>
```

### Actual result BEFORE applying this Pull Request
A modal opens and is empty.


### Expected result AFTER applying this Pull Request
A modal opens and shows a few entries


### Link to documentations
Please select:
